.github/labels.md

```diff
@@ -0,0 +1,238 @@
+Issue/PR occurs.  Categories share a prefix in the form
+`category_name:label_name` and are grouped together by color.  As a rule of
+thumb there is at least one label from every category that needs to be applied
+to an issue but use common sense to check if the label is appropriate or not for
+the specific Issue/PR.
+
+Currently there are three categories:
+
+
+#### Label category `category`
+
+Labels within this category describe the intent of the Issue/PR.  Usually there
+should be exact one intent for creating an Issue/PR and it should not change
+during the lifetime of an Issue/PR.
+
+#### Label category `component`
+
+The component lists the parts of the project this Issue/PR affects or is
+affected by.  This should help developers to identify if they have required
+knowledge or interests to handle the issue.  There should be at least one
+component listed for an Issue.
+
+#### Label category `status`
+
+With the status category developers, managers or issue reporters can check if
+an issue needs further interaction and who should carry out this interaction.
+An Issue/PR usually has exactly one status.  Status may change over the lifetime
+of the Issue/PR.
+
+
+## Labels
+
+This section lists when an label should be applied and what additional actions
+may be taken.
+
+
+#### Label `category:bug`
+
+The Issue/PR describes or solves a perceived malfunction within the game.
+
+#### Label `category:feature`
+
+The Issue/PR describes or implements a new game feature.  Features are parts of
+the game which a player can interact with e.g.: A new species, a new gameplay
+mechanic or a new command line switch for the server that changes gameplay
+behavior.
+
+#### Label `category:refactoring`
+
+The Issue/PR describes or contains a new or improved implementation of an
+existing feature or any other code restructuring that doesn't modify the current
+game mechanics or representation.
+
+#### Label `category:tweak`
+
+The PR contains insignificant code changes that don't involve features,
+refactoring or bug fixing but code style grooming or value tweaking.
+
+#### Label `component:AI`
+
+The Issue/PR deals with the Python AI decision making code or affects it.
+
+#### Label `component:art music text assets`
+
+The Issue/PR deals with creating or modifying game assets or verification if the
+assets match the style goals of the game.
+
+#### Label `component:build system`
+
+The Issue/PR deals with CMake, Visual Studio, Xcode, the respective platform
+SDKs or the general build process.
+
+#### Label `component:content scripting`
+
+The Issue/PR deals with the FOCS language, turn events, the universe generator
+and probably game mechanics.
+
+#### Label `component:deployment`
+
+The Issue/PR deals with the deployment process or the software used to create
+the installer or other distribution packages.
+
+#### Label `component:documentation`
+
+The Issue/PR deals with Doxygen or primarily contains documentation.
```

Comment:
If that label should only refer to technical documentation aimed at devs and contributors as you suggested, the description needs to clarify that.

We then also need an additional label for documentation for users/players. What do you think about the suggestions I made (about renaming the labels to convey their intended use more clearly, making it clear in the descriptions of the labels that it's not "external" vs. "internal/in-game pedia" documentation, but technical documentation vs. user manual)?